openreview/venue_request/process/recruitmentProcess.py

```diff
@@ -34,13 +34,22 @@ def process(client, note, invitation):
         'area chair': 'Area_Chairs',
         'senior area chair': 'Senior_Area_Chairs'
     }
+
+    # Fetch contact info
+    if conference.request_form_id:
```

Comment:
in the line 4 of this process function you already have `request_form` so no need to get the note again.